app/models/remote_authenticatable.rb

```diff
@@ -134,6 +133,8 @@ def serialize_from_session(data, _salt)
           resource.given_name = data['given_name']
           resource.family_name = data['family_name']
           resource.session_start_time = data['session_start_time']
+          resource.team = data['team']
+          resource.cognito_groups = data['cognito_groups'].split(',') unless data['cognito_groups'].nil?
```

Comment:
```suggestion
          resource.cognito_groups = data['cognito_groups']&.split(',')
```
You could make @jkunle happy! :D

Reply:
Made the change... though I personally prefer the unless as I think its more readable.